src/modules/Grayscale/Module.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Average all pixel colors
+ */
+module.exports = function Average(options, UI) {
+
+    options.blur = options.blur || 2
+    var output;
+
+    options.step.metadata = options.step.metadata || {};
+
+    function draw(input, callback, progressObj) {
+
+        progressObj.stop(true);
+        progressObj.overrideFlag = true;
+
+        var step = this;
+
+        function changePixel(r, g, b, a) {
+            return [r, g, b, a]
+        }
+
+        // do the averaging
+        function extraManipulation(pixels) {
+
+
+            for (var i = 0; i < pixels.data.length; i += 1) {
+                var avg = (pixels.data[i].getRed() + pixels.data[i].getGreen() + pixels.data[i].getBlue()) / 3;
```

Comment:
Are you about this usage of the API? I think it's like pixels.get(x,y,channel)

Reply:
Yes